packages/core/src/app/shipping/stripeUPE/StripeShipping.tsx

```diff
@@ -76,41 +76,31 @@ class StripeShipping extends Component<StripeShippingProps, StripeShippingState>
             isStripeAutoStep,
         } = this.state;
 
-            return <div className="checkout-form">
-                <div style={ {display: isStripeAutoStep ? 'none' : undefined,} }>
-                    <LoadingOverlay
-                        hideContentWhenLoading
-                        isLoading={ isStripeLoading }
-                    >
-                        <ShippingHeader
-                            isGuest={ isGuest }
-                            isMultiShippingMode={ isMultiShippingMode }
-                            onMultiShippingChange={ onMultiShippingChange }
-                            shouldShowMultiShipping={ shouldShowMultiShipping }
-                        />
-
-                        <LoadingOverlay
-                            isLoading={ isLoading }
-                            unmountContentWhenLoading
-                        >
-                            <StripeShippingForm
-                                { ...shippingFormProps }
-                                deinitialize={deinitialize}
-                                initialize={initialize}
-                                isBillingSameAsShipping={isBillingSameAsShipping}
-                                isLoading={ isLoading }
-                                isMultiShippingMode={isMultiShippingMode}
-                                isStripeAutoStep={this.handleIsAutoStep}
-                                isStripeLoading={this.stripeLoadedCallback}
-                                isShippingMethodLoading={isShippingMethodLoading}
-                                onSubmit={onSubmit}
-                                step={step}
-                                updateAddress={updateAddress}
-                            />
-                        </LoadingOverlay>
-                    </LoadingOverlay>
-                </div>
+        return <>
+            <AddressFormSkeleton isLoading={isStripeLoading}/>
+            <div className="checkout-form" style={{display: isStripeAutoStep || isStripeLoading ? 'none' : undefined}}>
```

Comment:
We don't need `style={{display: isStripeAutoStep || isStripeLoading ? 'none' : undefined}}` here because all skeleton components will display its children at the end of loading. 
Could we follow this usage? Thank you.
https://github.com/bigcommerce/checkout-js/blob/1f46f329e073226e333ba258f4dd2ba28543ff61/packages/core/src/app/shipping/Shipping.tsx#L159

Reply:
Hey @bc-peng, that’s why I had to do this (the child is being rendered at the end of loading ), the validations I use in the skeleton need the rendered component to be able to change, so, instead of using the skeleton component as it is, I just use it visually and at the same time I hide the content until be sure the StripeComponent is fully loaded.

Reply:
I think you can replace the condtion in the skeleton as below.
The appearance of the skeleton is controlled by `isLoading`.
```
<AddressFormSkeleton isLoading={isStripeAutoStep || isStripeLoading}/>
```

Reply:
Hey @bc-peng, I already tested the change, and makes sense, instead of showing blank microseconds we continue with the skeleton component, thanks :D